core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -203,6 +203,28 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
     getOption(key).map(_.toBoolean).getOrElse(defaultValue)
   }
 
+  // Limit of bytes for total size of results (default is 1GB)
```

Comment:
Hm, why is this property special-cased here? the methods in this class are generally generic.